reference_guide/api_changes_list_2023.md

```diff
@@ -100,6 +100,11 @@ JsonPath library unbundled
 `com.intellij.codeInsight.TailTypes` class renamed to `com.intellij.codeInsight.JavaTailTypes`
 : Update code usages.
 
+### JavaScript Plugin 2023.3
+
+`com.intellij.lang.javascript.buildTools.npm.PackageJsonUtil.createPackageJson(PsiDirectory, boolean) : PsiFile` method removed
```

Comment:
please remove part ` : PsiFile` after closing `)`

Reply:
Thank you so much. 
I'm so sorry. It's my first time 😇